File: src/main/java/com/binarysushi/studio/browseActions/StudioBrowseXChangeAction.java
package com.binarysushi.studio.browseActions;

import com.intellij.ide.BrowserUtil;
import com.intellij.openapi.actionSystem.AnAction;
import com.intellij.openapi.actionSystem.AnActionEvent;
import com.intellij.openapi.project.DumbAware;
import org.jetbrains.annotations.NotNull;

public class StudioBrowseXChangeAction extends AnAction implements DumbAware {

    @Override
    public void actionPerformed(@NotNull AnActionEvent e) {
        BrowserUtil.browse("https://xchange.demandware.com/");
    }
}

File: src/com/binarysushi/studio/browseActions/StudioBrowseXChangeAction.kt
package com.binarysushi.studio.browseActions

import com.intellij.ide.BrowserUtil
import com.intellij.openapi.actionSystem.AnAction
import com.intellij.openapi.actionSystem.AnActionEvent
import com.intellij.openapi.project.DumbAware

class StudioBrowseXChangeAction : AnAction(), DumbAware {
    override fun actionPerformed(e: AnActionEvent) {
        BrowserUtil.browse("https://xchange.demandware.com/")
    }
}
